LoopKitTests/TemporaryScheduleOverrideHistoryTests.swift

```diff
@@ -14,11 +14,14 @@ final class TemporaryScheduleOverrideHistoryTests: XCTestCase {
     // Midnight of an arbitrary date
     let referenceDate = Calendar.current.startOfDay(for: Date(timeIntervalSinceReferenceDate: .hours(100_000)))
 
-    let basalRateSchedule = BasalRateSchedule(dailyItems: [
-        RepeatingScheduleValue(startTime: .hours(0), value: 1.2),
-        RepeatingScheduleValue(startTime: .hours(6), value: 1.4),
-        RepeatingScheduleValue(startTime: .hours(20), value: 1.0)
-    ])!
+    let basalRateSchedule = BasalRateSchedule(
+        dailyItems: [
+            RepeatingScheduleValue(startTime: .hours(0), value: 1.2),
+            RepeatingScheduleValue(startTime: .hours(6), value: 1.4),
+            RepeatingScheduleValue(startTime: .hours(20), value: 1.0)
+        ],
+        timeZone: Calendar.current.timeZone
```

Comment:
This will have different values depending on where the test is run. Will the test fail in some zones?

Reply:
Well, without this it fails on my machine. Is there some magic I have to do to get this to work without the `timeZone` specified?

Reply:
We can specify timezone; it just should be test-machine independent. So maybe just create create a fixed offset timezone of -0600, or UTC

Reply:
Ah, the root problem is that code a few lines above, the `referenceDate` uses the current calendar, which uses the test machine's time zone. I'll update both to use GMT.

Reply:
Done.